examples/converters.py

```diff
@@ -0,0 +1,99 @@
+# This example requires the 'members' privileged intent to use the Member converter.
+
+import typing
+
+import discord
+from discord.ext import commands
+
+intents = discord.Intents.default()
+intents.members = True
+
+bot = commands.Bot("!", intents=intents)
+
+
+@bot.command()
+async def userinfo(ctx: commands.Context, user: discord.User):
+    # In the command signature above, you can see that the `user`
+    # parameter is typehinted to `discord.User`. This means that
+    # during command invocation, the `commands` framework will
+    # attempt to convert the value passed for the `user` parameter
+    # to a `discord.User` instance.
+
+    # NOTE: typehinting acts as a converter within the `commands` framework only.
+    # In standard Python, it is use for documentation and IDE assistance purposes.
+
+    # If the conversion is successful, we will have a `discord.User` instance
+    # and can do the following:
+    await ctx.send(user.mention)
+
+@userinfo.error
+async def userinfo_error(ctx: commands.Context, error: Exception):
+    # if the conversion above fails for any reason, it will raise `commands.errors.BadArgument`
```

Comment:
```suggestion
    # if the conversion above fails for any reason, it will raise `commands.BadArgument`
```

The explicit namespace is not part of the contract of the library.